src/staff_explode.lua

```diff
@@ -3,18 +3,20 @@ function plugindef()
     finaleplugin.Author = "Carl Vine"
     finaleplugin.AuthorURL = "http://carlvine.com"
     finaleplugin.Copyright = "CC0 https://creativecommons.org/publicdomain/zero/1.0/"
-    finaleplugin.Version = "v1.45"
-    finaleplugin.Date = "2022/05/16"
+    finaleplugin.Version = "v1.46"
+    finaleplugin.Date = "2022/07/09"
     finaleplugin.Notes = [[
         This script explodes a set of chords from layer one on one staff onto single lines on subsequent staves. 
         The number of staves is determined by the largest number of notes in any chord.
         It warns if pre-existing music in the destination will be erased. 
         It duplicates all markings from the original and resets the current clef on each destination staff.
 
-        This script allows for the following configuration:
-
+        By default this script respaces the selected music after running. 
+        To change this behaviour you must create a `configuration` file. 
+        If it does not exist, create a subfolder called `script_settings` in the folder containing this script. 
+        In that folder create a text file  called `staff_explode.config.txt` containing the line:  
         ```
-        fix_note_spacing = true -- to respace music automatically when the script finishes
+        fix_note_spacing = false
         ```
```

Comment:
I would add back the comment here:

```
fix_note_spacing = false -- to skip respacing music when the script finishes
```

Also, add a blank line between the last line of text and the three tick marks. Sometimes the documentation generator gets confused if there isn't a blank line.

Reply:
The lack of a blank line isn't causing an issue here. However, it's generally good practice to prevent ambiguous markdown.

Reply:
A neater and more useful variation - no user settings or config files. Just a simple second menu option. If this is approved I'll duplicate the process with my other 3 EXPLODE scripts.